README.rst

```diff
@@ -121,6 +121,20 @@ the categorical variables, but does *not* accept missing values for the
 numerical values. It is up to the user to come up with a way of
 handling these missing data that is appropriate for the problem at hand.
 
+Parallel execution
+------------------
+
+The k-modes and k-prototypes implementations both offer support for threaded
```

Comment:
The default behavior of `joblib.Parallel` is multi-processing over multiple cores, not multi-threading. Could make that clearer here?